src/App.js

```diff
@@ -1,26 +1,33 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
-import data from './api/movies.json';
 
-export class App extends Component {
-  state = {
-    movies: data,
-  };
+export const App = () => {
+  const [movies, setMovie] = useState([]);
+
+  const addMovie = (findedMovie) => {
+    setMovie((availableMovies) => {
+      const moviesForCompare = availableMovies
+        .filter(movie => movie.imdbID !== findedMovie.imdbID);
 
-  render() {
-    const { movies } = this.state;
+      if (availableMovies.length === moviesForCompare.length
+        && findedMovie.imdbID) {
+        return [...availableMovies, findedMovie];
+      }
```

Comment:
You can just check if there's SOME movie in the list using the array method 'some'